app-office/libreoffice/libreoffice-5.1.4.2.ebuild

```diff
@@ -265,7 +265,7 @@ pkg_pretend() {
 	if [[ ${MERGE_TYPE} != binary ]]; then
 		check-reqs_pkg_pretend
 
-		if [[ $(tc-getCC) == clang ]] ; then
+		if [[ $(tc-getCC) == *clang* ]] ; then
```

Comment:
It's been a week since I submitted a patch on this.  That's longer than a week that people have been having issues with it.  I am truly sorry, but I can't afford any more time on this.


I sincerely hope someone else fixes it.

On Wed, Aug 31, 2016 at 11:46:24AM -0700, Andreas Sturmlechner wrote:
> > @@ -265,7 +265,7 @@ pkg_pretend() {
> >  	if [[ ${MERGE_TYPE} != binary ]]; then
> >  		check-reqs_pkg_pretend
> >  
> > -		if [[ $(tc-getCC) == clang ]] ; then
> > +		if [[ $(tc-getCC) == *clang* ]] ; then
> 
> github is not just an alternative place to bugzilla to voice your concerns, this is where you get involved yourself to help. The (already existing) bug is enough to make maintainers aware of the issue, if that's what you want.
> 
> -- 
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub:
> https://github.com/gentoo/gentoo/pull/2145/files/6992b9b78310bb8769a490a6deeaee1bf23aeb45#r77048658